README.md

```diff
@@ -5,8 +5,9 @@ The applications in this repo are all very similar in their `main.rs`. The purpo
 - `stm32h745-defmt-rtt`: Similar to above, except it uses `defmt` for RTT
 - `nrf52833`: A sample applciation for Cortex-M4 on a Micro:Bit v2 board
 - `esp32c3` : A sample configuration for RISCV platform
+- `pico`    : A sample configuration for Cortex-M0 on Raspberry PICO RP2040, that uses a second PICO as a probe.
 
-To customize or create additional application, simply copy the contents of a similar architecture, and look for the `TODO:` comment tag in all the files to identify the changes needed to support a specific architecture.
+To customize or create additional application, simply copy the contents of a similar architecture, and look for the `TODO:` comment tag in all the files to identify the changes needed to support a specific architecture or board.
```

Comment:
```suggestion
To customize or create an additional application, simply copy the contents of a similar architecture, and look for the `TODO:` comment tag in all the files to identify the changes needed to support a specific architecture or board.
```

Also, maybe a `REPLACE` or `FILL` tag would be better as `TODO` is normally used for a code task which could also be meant as "Template code has this todo" :) It's not a blocker tho!

Reply:
Thanks. yes, I will do that next time around. I'm actively working on this ... I wish there was an easier way to template code (.toml, .json and .rs) for different development boards. Do you know of any?

Reply:
Latest version I used CONFIGURE